lib/rest/Twilio.d.ts

```diff
@@ -146,13 +146,15 @@ declare namespace Twilio {
    * Options to pass to the Twilio Client constructor
    *
    * @property accountSid - The default accountSid. This is set to username if not provided
+   * @property edge - Twilio region to use. Defaults to none
```

Comment:
`s/region/edge/`

Reply:
This has not been resolved.

Reply:
Sorry, I made these changes in Yoyodyne and have not applied them yet.